macros/edr/system/system_utils/logs.sql

```diff
@@ -9,6 +9,8 @@
         {% set debug_logs_enabled = elementary.get_config_var('debug_logs') %}
         {% if debug_logs_enabled %}
             {{ elementary.edr_log(msg) }}
+        {% else %}
+            {{ elementary.edr_log(msg, info=False) }}
```

Comment:
Are we sure about it? is it going to slow down our performance? this means all of our debug logs will be printed ot the log file right? not sure about it in terms of perf